Cargo.toml

```diff
@@ -27,7 +27,7 @@ rand-std = ["rand/std"]
 recovery = ["secp256k1-sys/recovery"]
 lowmemory = ["secp256k1-sys/lowmemory"]
 global-context = ["std", "rand-std", "global-context-less-secure"]
-global-context-less-secure = []
+global-context-less-secure = ["std"]
```

Comment:
I'd rather we actually fix the build rather than depend on std (see conversation at https://github.com/rust-bitcoin/rust-secp256k1/pull/359 for one way to do this). The reason for `global-context-less-secure`, in part, is that we want to support `global-context` without the `rand` dependency, in part for platforms where we can't use `rand`. There are some platforms (like wasm, specifically), where we can technically build without no-std, but where we may not actually have an actual std environment to call, relying on LTO to remove the calls that we can't make.

Reply:
Cool, I'm off for the next few days. I'll convert this to draft and re-spin when I'm back. Thanks.

Reply:
Now I have read that link I think this whole PR should be closed.